specs/BadgeNotifications/BadgeNotifications-spec.md

```diff
@@ -0,0 +1,213 @@
+
+For more details see:
+
+-   [Badge Notification WinRT APIs](https://learn.microsoft.com/en-us/windows/apps/design/shell/tiles-and-notifications/badges)
+    Defines all the API constructs that we have for Badge Notifications in WinRT today.
+
+# Examples
+
+## Create a numeric badge (c++)
+
+```c++
+private void setBadgeNumber(uint32_t num)
+{
+   winrt::BadgeNotification badgeNotification(num);
+   winrt::BadgeNotificationManager manager = winrt::BadgeNotificationManager::Current();
+   manager.Update(badgeNotification);
+}
+```
+
+## Create a Glyph badge (c++)
+
+```c++
+private void updateBadgeGlyph()
+{
+   winrt::BadgeNotificationGlyph badgeNotificationGlyph = winrt::BadgeNotificationGlyph::Alert
+   winrt::BadgeNotification badgeNotification(badgeNotificationGlyph);
+   winrt::BadgeNotificationManager manager = winrt::BadgeNotificationManager::Current();
+   manager.Update(badgeNotification);
+}
+```
+
+## Clear a badge
+
+```c++
+private void clearBadge()
+{
+   winrt::BadgeNotificationManager manager = winrt::BadgeNotificationManager::Current();
+   manager.Clear();
+}
+```
+
+## To set an expiration time in a badge (c++)
+
+```c++
+private void updateBadgeGlyph()
+{
+   winrt::BadgeNotificationGlyph badgeNotificationGlyph = winrt::BadgeNotificationGlyph::Alert
+   winrt::BadgeNotification badgeNotification(badgeNotificationGlyph);
+
+   DateTime expirationTime = clock::now() + std::chrono::hours(24);
+   badgeNotification.Expiration(expirationTime);
+
+   winrt::BadgeNotificationManager manager = winrt::BadgeNotificationManager::Current();
+   manager.Update(badgeNotification);
+}
+```
+
+# API Description
+
+## BadgeNotification class
+
+The `BadgeNotification` class is used to create badge notifications for apps on platforms such as
+Windows, where badges can be displayed on the app's tile. A badge can show a numeric value or a
+system-provided glyph, providing a visual indicator of some status or count directly on the app's
+icon.
+
+## BadgeNotification constructors
+
+| Name                                      | Description                                                                                                                                                                              |
+| ----------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| BadgeNotification(UInt32)                 | Creates a new `BadgeNotification` object with a numeric badge value. This is used to display a specific number on the app's badge.                                                       |
+| BadgeNotification(BadgeNotificationGlyph) | Creates a new `BadgeNotification` object with a glyph value. This uses a predefined glyph from the `BadgeNotificationGlyph` enum to display a system-provided symbol on the app's badge. |
```

Comment:
Remind me, are overloads with the same arity OK in WinAppSDK?  I remember this causes problems for projections into weakly typed languages.

Reply:
BadgeNotification class has been removed from the Spec as per discussion